test/collins/util/parsers/LshwParserSpec.scala

```diff
@@ -492,5 +492,18 @@ class LshwParserSpec extends mutable.Specification {
       }
     } // wonky opterons
 
+    "Parse GPU server" in {
```

Comment:
I'm working on this, but it will probably have to be a separate PR.

We (and by "we" I mean @michaeljs1990) found that our lshw XML files broke all of the existing model tests in `LshwHelperSpec`. In order to get the new GPU code to pass the tests we had to "add" GPUs to the existing XML test files.

Unfortunately it looks like there is something unique about our hardware. But with the exception of the model tests, everything else seems to work fine with our lshw XML data.